config/zones.json

```diff
@@ -383,12 +383,21 @@
   },
   "CA-QC": {
     "_comment": "http://www.hydroquebec.com/generation/",
+    "_comment": "http://www.hydroquebec.com/decom",
```

Comment:
You can't have duplicated keys

Reply:
removed - no CA_QC.py parser (or in progress) to move to - will save info offline